CHANGELOG.md

```diff
@@ -81,6 +81,7 @@
 
 - Refactored `quoted_identifier_to_snowflake_type` to avoid making metadata queries if the types have been cached locally.
 - Improved `pd.to_datetime` to handle all local input cases. 
+- Refactored `resample.fillna` implementation to use ASOF Join.
```

Comment:
actually, that is just an internal refactoring,  which has no impact to users, let's remove the CHANGELOG here